### ISSUE TITLE
Benchmark wanted: tail latency in http requests

### ISSUE THREAD

**Issue by ry** [denoland/deno#1963]
During [the http_benchmarks](https://github.com/denoland/deno/blob/master/tools/http_benchmark.py) we run [wrk](https://github.com/wg/wrk). It outputs a small table that looks like this:
```
Running 10s test @ http://127.0.0.1:4500/
  2 threads and 10 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency     5.90ms   29.21ms 425.93ms   95.69%
    Req/Sec    16.55k     3.08k   22.18k    83.17%
  332652 requests in 10.10s, 16.18MB read
Requests/sec:  32931.55
Transfer/sec:      1.60MB
```

Currently we only extract the requests per second (second to last line). [It is plotted here](https://deno.land/benchmarks.html#req-per-sec). 

I also want to keep track of the "max latency" number (425.93ms in the above example). This can be extracted at the same time the "req-per-sec" benchmark is run. A new plot should be added to [the benchmark page](https://github.com/denoland/deno/blob/master/website/benchmarks.html).

**Comment by bernard-lin** [comment]
I'll take a crack at this

**Comment by bartlomieju** [comment]
Resolved in #1975
